README.md

```diff
@@ -148,6 +148,10 @@ for all files in `src` directory that match the glob pattern
 To package your app into an installer use command:
 
 ```sh
+npm install --production
+$env:NODE_ENV="production"
```

Comment:
This line here is specific to PowerShell (Windows) users. Someone building from `cmd` or from other platform+shell combo (e.g. zsh over macOS) will find this line confusing.